src/test-run/index.ts

```diff
@@ -775,6 +781,168 @@ export default class TestRun extends AsyncEventEmitter {
         return consoleMessageCopy[String(this.activeWindowId)];
     }
 
+    public async _enqueueGetCookies (command: CommandBase): Promise<unknown> {
+        const cookiesPromises = [];
+
+        const cookies = command.cookies as any;
+
+        const names = command.names as string[];
+        const urls  = command.urls as string[];
+
+        if (cookies) {
+            for (const cookie of cookies) {
+                const domain = cookie.domain;
+                const path   = cookie.path;
+                const name   = cookie.name;
+
+                if (domain) {
+                    if (name)
+                        cookiesPromises.push(this.session.cookies.findCookieByApi(domain, path, name));
```

Comment:
Can we change signature of `session.cookies.findCookieByApi` function to allowing it asset an array of arguments?